Examples/Tutorials/Tutorial-DpdkL2Fwd/main.cpp

```diff
@@ -68,49 +52,46 @@ int main(int argc, char* argv[])
 	pcpp::DpdkDeviceList::initDpdk(coreMaskToUse, MBUF_POOL_SIZE);
 
 	// Find DPDK devices
-	pcpp::DpdkDevice* device1 = pcpp::DpdkDeviceList::getInstance().getDeviceByPort(DEVICE_ID_1);
-	if (device1 == NULL)
+	auto* device1 = pcpp::DpdkDeviceList::getInstance().getDeviceByPort(DEVICE_ID_1);
+	if (device1 == nullptr)
 	{
-		std::cerr << "Cannot find device1 with port '" << DEVICE_ID_1 << "'" << std::endl;
+		std::cerr << "Cannot find device1 with port '" << DEVICE_ID_1 << "'\n";
```

Comment:
isn't using `std::endl` clearer? In addition, we need to support different platforms, using `std::endl` is more compatible.